Unresolved Import `http`

Received this error on a brand new project created today.
```
error[E0432]: unresolved import `http`
 --> /Users/username/.cargo/registry/src/github.com-1ecc6299db9ec823/create-rust-app-6.0.1/src/util/actix_web_utils.rs:3:5
  |
3 | use http::StatusCode;
  |     ^^^^ help: a similar path exists: `actix_web::http`
```

Cargo.toml
```
[[bin]]
name = "fullstack"
path = ".cargo/bin/fullstack.rs"

[[bin]]
name = "tsync"
path = ".cargo/bin/tsync.rs"

[[bin]]
name = "my-project"
path = "backend/main.rs"

[dependencies]
actix-files = "0.6.0"
actix-http = "3.0.0"
actix-multipart = "0.4.0"
actix-web = "4.0.1"
async-graphql = "3.0.38"
futures-util = "0.3.21"
jsonwebtoken = "8.1.0"
async-graphql-actix-web = "3.0.38"
serde_json = "1.0.79"
tsync = "1.2.1"

[dependencies.chrono]
features = ["serde"]
version = "0.4.19"

[dependencies.create-rust-app]
features = ["plugin_dev", "plugin_auth", "plugin_container", "plugin_graphql", "backend_actix-web"]
version = "6.0.1"

[dependencies.diesel]
default-features = false
features = ["postgres", "r2d2", "chrono"]
version = "1.4.8"

[dependencies.serde]
features = ["derive"]
version = "1.0.133"

[dependencies.tokio]
features = ["full"]
version = "1"

[package]
default-run = "my-project"
edition = "2021"
name = "my-project"
publish = false
version = "0.1.0"
[profile.dev]
debug-assertions = true

```

```
cargo --version
cargo 1.60.0 (d1fd9fe2c 2022-03-01
```

--- a/create-rust-app/src/util/actix_web_utils.rs
+++ b/create-rust-app/src/util/actix_web_utils.rs
@@ -1,6 +1,6 @@
 use actix_files::NamedFile;
 use actix_web::{HttpRequest, HttpResponse, Scope, web};
-use http::StatusCode;
+use actix_web::http::StatusCode;
 use tera::Context;
 use crate::util::template_utils::TEMPLATES;
 

